pkg/accesscontrol/grantee.go

```diff
@@ -140,16 +143,16 @@ func NewGranteeListReference(ctx context.Context, ls file.LoadSaver, reference s
 	}, nil
 }
 
-func serialize(publicKeys []*ecdsa.PublicKey) []byte {
+func serialize(publicKeys []*ecdsa.PublicKey) ([]byte, error) {
 	b := make([]byte, 0, len(publicKeys)*publicKeyLen)
 	for _, key := range publicKeys {
-		b = append(b, serializePublicKey(key)...)
+		// TODO: check if this is the correct way to serialize the public key
+		// Is this the only curve we support?
+		// Should we have switch case for different curves?
+		pubBytes := crypto.S256().Marshal(key.X, key.Y)
```

Comment:
Here, we only have public key, and we could have something like this:
```
pubKey, err := key.ECDH()
if err != nil {
	return nil, fmt.Errorf("generate key: %w", err)
}
b = append(b, pubKey.Bytes()...)
```
But unit tests will return: `unsupported curve by crypto/ecdh`. Because of this I used specific curve that we are also using in `NewEthereumAddress` function.

Reply:
Is this comment still relevant?